pkg/csv/ingest.go

```diff
@@ -77,7 +82,8 @@ func (ingester Ingester) Ingest(data []string, ID string) (ls.Node, error) {
 				name:  propertyName,
 				value: columnData,
 			})
-		} else {
+		} else if ingester.OnlySchemaAttributes {
```

Comment:
It should ingest the node if schemaNode is not nil, or if schemaNode is nil and ingester.OnlySchemaAttributes is set. So this should be:

if schemaNode!=nil || ingester.OnlySchemaAttributes {... }